torch/distributed/distributed_c10d.py

```diff
@@ -1958,6 +1972,104 @@ def destroy_process_group(group: Optional[ProcessGroup] = None):
         _unregister_process_group(pg.group_name)
 
 
+def _abort_process_group(group: Optional[ProcessGroup] = None):
```

Comment:
This is a public API right? so we should name it abort_process_group without '_'?

Reply:
It should be public, but also want to keep private for a little bit till mature.

Reply:
OK, maybe we should limit its usage to only certain NCCL versions?